### DISCUSSION TITLE
Succeful - Failed Table to Add to speedtest-tracker

### DISCUSSION THREAD

**Discussion by masterwishx** [alexjustesen/speedtest-tracker#569] [Ideas]
is any chance to add Succeful - Failed Table from old speedtest-tracker to this new one ?

![image](https://user-images.githubusercontent.com/28630321/235338513-7658d22b-d1c4-4e42-ac42-697799d67166.png)

**Comment by alexjustesen** [comment]
On the list for sure, I think it's already captured in an open issue
